shared/actions/teams.js

```diff
@@ -318,15 +336,7 @@ const addToTeam = (_, action) => {
     addToTeamWaitingKeys(teamname, username)
   )
     .then(() => {})
-    .catch(e => {
-      // identify error
-      if (e.code === RPCTypes.constantsStatusCode.scidentifysummaryerror) {
-        if (isMobile) {
-          // show profile card on mobile
-          return ProfileGen.createShowUserProfile({username})
-        }
```

Comment:
Not sure what to do for desktop here, so just made it pop up a tracker. cc @mlsteele who wrote this code originally -- what should we do here?

Reply:
This is an improvement now there's feedback :)
I just coped what addToTeam did. I think the ideal thing here would be something that guides the user through their options. Which are either to cancel or to re-follow the person and forge forward. So that would be tracker screen (not popup) with "Cancel" and "Accept changes + Add" buttons. The second would follow and retry the add.

But that seems like a lot of work. So this is fine.

Reply:
I didn't try this on mobile but I assume it would be a surprise that after you click "Accept" on the tracker, they don't actually get added to the team. But the re-admit button is still in your face at that point.

Reply:
Ah, I see! Looking through addToTeam's history, I found https://github.com/keybase/client/pull/14915 which says that desktop was intended to pop up a tracker anyway. so I think this is fine for the release, and we can do something more complicated later.